### ISSUE TITLE
[Feature|Bug|Improvement] DDEV list table overflows

### ISSUE THREAD

**Issue by Firesphere** [ddev/ddev#5535]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

I started briefly on PR #5534 , however, as mentioned by @rfay , the width of the table already causes issues with slimmer terminal settings.

Adding the 3 characters required to fit "silverstripe" increases the chances of the table overflowing.

See this table:
```
┏━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━┓
┃ NAME                     ┃ STATUS  ┃ LOCATION                            ┃ URL                    ┃ TYPE      ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━╋━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━━━┫
┃ proj1                    ┃ stopped ┃ ~/Projects/proj1                    ┃                        ┃ silverstr ┃
┃                          ┃         ┃                                     ┃                        ┃ ipe       ┃
┃ proj2                    ┃ stopped ┃ ~/Projects/proj2                    ┃                        ┃ php       ┃
```
It's specifically/only Silverstripe projects that overflow at the moment.

### Describe your solution

Possible solutions:

1. Accept overflow, widen Type to 12 characters.
2. Slim down the project type names.
3. Find other places to reduce the width of the table.

## Option 1

No solution besides widening the table and accept the implications

## Option 2

Wordpress (The second-longest supported framework) can be slimmed down to `WP`, but slimming down Silverstripe creates the awkward problem of the abbreviation being "SS"

An option for Silverstripe, would be to change the name in the table to `SSCMS` (Short for Silverstripe CMS). 
Further, if Wordpress would be abbreviated to "WP", the type table could be limited to 7 characters (Laravel).

### Status column

Other places where the width of the table could be adjusted are limited. The only one I could think of, was in the Status column, instead of the words fully written out.
E.g.:
`stopped` => `stop`
`running` => `run`
`paused` => `pause`
This would give 2 characters of "space" compared to the fully written out, however, the word "STATUS" is already 6 characters, thus making it only 1.

### URL column

The URL could be slimmed down by removing `https://`, however, it doesn't solve the problem, since the length of the URL is already variable.

**Comment by tyler36** [comment]
Is there a "truncate" option?

**Comment by tyler36** [comment]
Yes, not sure if its available but it would make things more dynamic and future-proof.
It could be applied to the "TYPE", but it could also be applied to "STATUS"

**Comment by rfay** [comment]
The other thing is to spend some time messing with the underlying library ([github.com/jedib0t/go-pretty/v6/table](https://github.com/jedib0t/go-pretty/)). Sometimes they make some progress, and maybe they've made some progress with this. 

These two issues are closed, https://github.com/jedib0t/go-pretty/issues?q=is%3Aissue+author%3Arfay+is%3Aclosed - maybe something good happened.

**Comment by rfay** [comment]
Would dearly love you to experiment with this and look at solutions @Firesphere

**Comment by Firesphere** [comment]
> Would dearly love you to experiment with this and look at solutions @Firesphere

Sorry! Completely slipped my mind as some things in life took over! I've put it back on my list of things to have a look at!
